examples/jsm/nodes/accessors/Object3DNode.js

```diff
@@ -99,6 +99,8 @@ class Object3DNode extends Node {
 
 		}
 
+		this._uniformNode.construct( builder );
```

Comment:
I tried to move this to Object3DNode.construct(), but it seems like it is never executed?..